update to android studio sample

oalami opened the pull request:
@lesv please review and merge

Commit:
add .gitignore

Commit:
update ignore file

Commit:
inital move to android studio and gradle

Commit:
CONTRIB.md -> CONTRIBUTING.md

Commit:
update text of contributing

Commit:
update to latest gradle version, regenerated endpoints

lesv commented on app/build.gradle:
```diff
@@ -0,0 +1,28 @@
+
+    defaultConfig {
+        applicationId "com.google.devrel.samples.ttt"
+        minSdkVersion 14
```
These days you should be build 17 as min. (Lowest Android Team recommends supporting)

lesv commented on app/build/generated/source/buildConfig/androidTest/debug/com/google/devrel/samples/ttt/test/BuildConfig.java:
```diff
@@ -0,0 +1,13 @@
+/**
```
Not sure build/generated should be checked in (et al)

lesv commented:
PTAL

Commit:
remove build files from source control

Commit:
update gitignore, remove intellij file

Commit:
update ignore file

Commit:
update android min ver

oalami commented:
@lesv ugh, shouldn't try to send a PR right before dinner. Removed the build files + updated the min version, back to you.

lesv commented:
I also sent you a suggested .gitignore

On Wed, Mar 25, 2015 at 9:13 PM, Ossama Alami <notifications@github.com>
wrote:

> @lesv <https://github.com/lesv> ugh, shouldn't try to send a PR right
> before dinner. Removed the build files + updated the min version, back to
> you.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/appengine-endpoints-tictactoe-android/pull/6#issuecomment-86332228>
> .
>



-- 
Les Vogel | Cloud Developer Relations | lesv@google.com | 408-676-7023

Commit:
update ignore file

oalami commented:
updated.

On Wed, Mar 25, 2015 at 9:21 PM, Les Vogel <notifications@github.com> wrote:

> I also sent you a suggested .gitignore
>
> On Wed, Mar 25, 2015 at 9:13 PM, Ossama Alami <notifications@github.com>
> wrote:
>
> > @lesv <https://github.com/lesv> ugh, shouldn't try to send a PR right
> > before dinner. Removed the build files + updated the min version, back to
> > you.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/GoogleCloudPlatform/appengine-endpoints-tictactoe-android/pull/6#issuecomment-86332228
> >
> > .
> >
>
>
>
> --
> Les Vogel | Cloud Developer Relations | lesv@google.com | 408-676-7023
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/appengine-endpoints-tictactoe-android/pull/6#issuecomment-86336102>
> .
>



-- 
Ossama Alami | Google Developer Relations | oa@google.com

lesv commented:
LGTM - If you had merged the repos (Android & GAE), you wouldn't need to checkin the generated code.  They would just generate it when needed.  We had Antoine also merge the iOS so that folks could look just one place and get the right thing.  That said, LGTM, if you feel inspired great, if not, just merge, or ask again and I can.

oalami commented:
doesn't look as I have write access. Will figure out why tomorrow, in the meantime, please merge.
